### PR TITLE
feat(links): add CTA variant - INNO-1580

### PR DESCRIPTION
**Pull request by yhuard**
This PR adds a new variant to the link component: "CTA" (call-to-action). It looks like the CTA button, except that the link is underlined. Both hero and page banners use this new variant.

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 8ce2d4b006ca272aafe8843aedc60e016b369582

https://deploy-preview-1176--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link.scss [hunk 1]
```diff
@@ -69,6 +69,34 @@
     }
     /* stylelint-enable */
   }
+
+  .ecl-link--cta {
```
Comment (emeryro) [inline src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link.scss:73]: the button uses class `ecl-button--call`. We should keep some consistency (`ecl-link--call`)

File: src/systems/eu/implementations/vanilla/packages/eu-component-link/eu-component-link.scss [hunk 2]
```diff
@@ -69,6 +69,34 @@
...
+
+  .ecl-link--cta {
+    background-color: $ecl-color-yellow-100;
+    box-sizing: border-box;
+    color: $ecl-color-black-100;
+    display: inline-block;
+    font: $ecl-font-m;
+    font-weight: $ecl-font-weight-bold;
+    padding: $ecl-spacing-s $ecl-spacing-m;
+    text-decoration: underline;
```
Comment (emeryro) [inline src/systems/eu/implementations/vanilla/packages/eu-component-link/eu-component-link.scss]: this is already defined in ecl-link, no need to duplicate (same for hover and outline in focus)

**Review by emeryro — CHANGES_REQUESTED** [review]
almost good!

File: src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link.scss (hunk 1 above)
Comment (yhuard) [reply src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link.scss:73]: I disagree :) 
1. a link is not a button, there's no reason to maintain consistency between them
2. `cta` is much clearer than `call`

Comment (emeryro) [reply src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link.scss:73]: ok, as agreed, we will keep it with `cta` here and update the buttons to use the same name

**Review by emeryro — APPROVED** [review]

**Pull request merged**